README.md

```diff
@@ -1,6 +1,6 @@
 # StableHLO
 
-StableHLO is an operation set for high-level operations (HLO) in machine
+StableHLO is an operator set for high-level operations (HLO) in machine
```

Comment:
The usual way of referring to StableHLO is "operation set". Both in [the spec](https://github.com/openxla/stablehlo/blob/main/docs/spec.md) and in [the MLIR language reference](https://mlir.llvm.org/docs/LangRef/#operations) that we're drawing inspiration from, ops are called operations, not operators. Let's use the more conventional terminology here as well?

Reply:
Oops. "Operator" ended up in our draft for the landing page and I thought that was from you so I copied it here. Reverted.